source/less/blocks/page.less

```
@@ -8,10 +8,29 @@
 }
 
 .page__body {
+  position: relative;
   min-height: 100%;
   margin: 0;
   display: grid;
   grid-template-rows: min-content 1fr min-content;
   color: @special-dark-grey;
   background-color: @basic-white;
+
+  &--contrast-background {
+    background-image: linear-gradient(@status-error 309px, @cover-green 310px, @cover-green 612px, @status-error 612px, @status-error 100%), url("../../img/index-bg-promo.png");
+    background-repeat: no-repeat;
+    background-position: 0 0, center 309px;
+    background-size: 100% 100%, cover;
```

```suggestion
    background-position: 50% 264px;
    background-image:
      linear-gradient(to bottom, @cover-green 302px, @basic-white 302px),
      url('../../img/index-background-desktop-2.png');
    background-repeat: no-repeat;
```

264px - это высота раскрытого меню, поэтому эту цифру надо будет тут писать, когда у меню есть класс открытости.
Так там 66px - это высота хедера на мобилке.
66 точно надо сделать переменной, тк ты используешь её в хедере (только там не мин-высота, а просто высота, хедер не может быть больше 66пк на мобилке)